rllib/agents/ddpg/ddpg_policy.py

```diff
@@ -74,6 +74,8 @@ def postprocess_trajectory(self,
 
 class DDPGTFPolicy(DDPGPostprocessing, TFPolicy):
     def __init__(self, observation_space, action_space, config):
+        self.observation_space = observation_space
```

Comment:
This will all be cleaned up properly and no longer necessary once we move DDPG into the functional Trainer API.